README.md

```diff
@@ -31,7 +31,34 @@ The repository for this project is one method where you can monitor progress - w
 
 # &#127939; How do I get started?
 
-## Embedding pipeline
+## Data Scraping pipeline
+
+Before we start running the pipeline, an additional setup for creating a storage container on Azure is required to store the data. We use [DVC](https://dvc.org/doc/user-guide/data-management/remote-storage/azure-blob-storage) tool for data versioning and data management.
+
+> Data version [documentation](docs/data-version-control.md) guides you through the process of setting up a storage container on Azure and configuring DVC.
+
+Run the data scraping pipeline.
+
+```bash
+python run.py --scrape
+```
+
+In this pipeline, there are two steps:
+
+* Scrape data from Mind and NHS Mental Health websites
+* Store and version the scraped data in a storage container on Azure using DVC
+
+## Data Preparation pipeline
+
+Run the data preparation pipeline.
```

Comment:
Now that we have data scraped, we're ready to prepare that data for the model. We've created a separate pipeline for this, where we clean, validate, and version the data...

To run the data preparation pipeline: